tests/base.py

```diff
@@ -124,6 +124,15 @@ def setUp(self, config_dict='config_dict'):
       'attributes': [{
         'key': 'test_attribute',
         'id': '111094'
+      }, {
+        'key': 'boolean_attribute',
```

Comment:
it is key not attribute, please make it boolean_key.

Reply:
same comments for below

Reply:
Okay. I will change it. In my opinion, boolean key gave a different meaning.